pyproject.toml

```diff
@@ -31,6 +31,9 @@ tm-devices = "^1.1.0"
 watchfiles = "^0.21.0"
 keyrings-cryptfile = "^1.3.9"
 httpx = "^0.26.0"
+typer = "^0.9.0"
+rich = "^13.7.0"
+python-frontmatter = "^1.1.0"
```

Comment:
No need to add them into the build
To install them: `poetry install --with dev`

Reply:
Didn't mean to commit this, thanks for noticing